### PR TITLE
Imp 1 maven support

### PR DESCRIPTION
**Pull request by SKART1**
#11 try to fix

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>se.vidstige</groupId>
+    <artifactId>jadb</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <url>https://github.com/adelolmo/jadb</url>
```
Comment (vidstige) [inline pom.xml]: this should be "https://github.com/vidstige/jadb", right?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+    <scm>
+        <url>scm:git:git@github.com:adelolmo/jadb.git</url>
```
Comment (vidstige) [inline pom.xml]: here as well?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        <connection>scm:git:git@github.com:adelolmo/jadb.git</connection>
```
Comment (vidstige) [inline pom.xml]: ...and here

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        <tag>HEAD</tag>
+    </scm>
+
+    <distributionManagement>
+        <repository>
+            <id>bintray-andonidelolmo-main</id>
+            <name>andonidelolmo-main</name>
+            <url>https://api.bintray.com/maven/andonidelolmo/main/jadb/;publish=1</url>
+        </repository>
+    </distributionManagement>
```
Comment (vidstige) [inline pom.xml]: what does this section do? Can we remove it?

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+    <scm>
+        <url>scm:git:git@github.com:adelolmo/jadb.git</url>
+        <connection>scm:git:git@github.com:adelolmo/jadb.git</connection>
+        <developerConnection>scm:git:git@github.com:adelolmo/jadb.git</developerConnection>
```
Comment (vidstige) [inline pom.xml]: ...and here.

**Comment by vidstige** [comment]
Are the unit tests run as part of building somehow here? Just asking.

Some places reference adelmo or somthing, which I guess should be changed before merge. Otherwise looks very good. Thank you for your contribution! 👍

File: pom.xml (hunk 1 above)
Comment (SKART1) [inline pom.xml]: Sorry - I have pushed too early version without actually restoring the author

File: pom.xml (hunk 2 above)
Comment (SKART1) [inline pom.xml]: Sorry - I have pushed too early version without actually restoring the author

File: pom.xml (hunk 3 above)
Comment (SKART1) [inline pom.xml]: Sorry - I have pushed too early version without actually restoring the author

File: pom.xml (hunk 4 above)
Comment (SKART1) [inline pom.xml]: Already removed. But actually it allows to publish your artifact to some of the repositories with only one command from command line. As your project is not in repository (which is actually sad =) ) this block is removed

File: pom.xml (hunk 5 above)
Comment (SKART1) [inline pom.xml]: corrected

**Comment by vidstige** [comment]
Looks great, but now there is a merge conflict :-/

**Comment by vidstige** [comment]
You know how to deal with that? It's fairly straight-forward.

**Comment by SKART1** [comment]
Conflict resolved. Just fixed the same test case as was already fixed

**Pull request merged**